modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/ViewBuilderScreen/ViewBuilderScreen.tsx

```diff
@@ -49,12 +51,14 @@ const ViewBuilderScreen: React.FC<{}> = () => {
 		const parentWindow = Liferay.Util.getOpener();
 
 		parentWindow.Liferay.fire('openModalAddColumns', {
-			getName: ({label}: ObjectField) => label[defaultLanguageId],
+			getName: ({label}: ObjectField) =>
+				getLocalizableLabel(label, creationLanguageId),
```

Comment:
@alinedoleron we can extract the `name` from `ObjectField` and use as the `fallback` here.